fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,142 @@
+# import datetime
+import json
+
+# import yaml
+import time
+from typing import Any
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from datadog_checks.base.utils.db import QueryManager
+# from requests.exceptions import ConnectionError, HTTPError, InvalidURL, Timeout
+# from json import JSONDecodeError
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
```

Comment:
Is this supposed to be visible to everyone?

Reply:
@yzhan289 - no. Again this is just for a test. Later on I will have to bring this into a config file that the customer will update for each integration. Can you point me on how I can read these values from a config file. Whats the preferred approach for the DD agent for reading these integration variables?

Reply:
Answered in other comment.